index.html

```diff
@@ -650,88 +682,118 @@ <h3>JSON Examples</h3>
-  ],
-  "verificationMethod": [
-    {
-      "id": "#urn:ietf:params:oauth:jwk-thumbprint:sha-256:NzbLsXh8uDCcd-6MNwXF4W_7noWXFZAfHkxZsRGC9Xs",
-      "type": "JsonWebKey",
-      "controller": "did:web:vendor.example",
-      "publicKeyJwk": {
-        "kty": "EC",
-        "crv": "P-521",
-        "alg": "ES512",
-        "x": "AFTyMw-fIYJNg6fBVJvOPOsLxmnNj8HgqMChyRL0swLaefVAc7wrWZ8okQJqMmvv03JRUp277meQZM3JcvXFkH1v",
-        "y": "ALn96CrD88b4TClmkl1sk0xk2FgAIda97ZF8TUOjbeWSzbKnN2KB6pqlpbuJ2xIRXvsn5BWQVlAT2JGpGwDNMyV1"
-      }
-    },
-    {
-      "id": "#z6MkhEdpG12jyQegrr62ACRmNY8gc531W2j9Xo39cHphuCEH",
-      "type": "JsonWebKey2020",
-      "controller": "https://vendor.example",
-      "publicKeyJwk": {
-        "kid": "urn:ietf:params:oauth:jwk-thumbprint:sha-256:FfMbzOjMmQ4efT6kvwTIJjelTqjl0xjEIWQ2qobsRMM",
-        "kty": "OKP",
-        "crv": "Ed25519",
-        "alg": "EdDSA",
-        "x": "ANRjH_zxcKBxsjRPUtzRbp7FSVLKJXQ9APX9MP1j7k4"
-      }
-    }
-    {
-      "id": "#subject-authenticaton",
-      "type": "JsonWebKey",
-      "controller": "did:web:vendor.example",
-      "publicKeyJwk": {
-        "kty": "EC",
-        "crv": "P-384",
-        "alg": "ES384",
-        "x": "PxgAmVYOQvSNcMYL2tOzoLwSWn4Ta3tIMPEUKR8pxeb-gmR11-DyKHBoIiY-2LhM",
-        "y": "BZEBTkImVdpwvxR9THIRw16eblnj5-tZa7m-ww5uVd4kyPJNRoWUn2aT9ZuarAe-"
-      }
-    },
+  "@context": ["https://www.w3.org/ns/credentials/v2"],
+  "type": ["VerifiablePresentation"],
+  "holder": "urn:ietf:params:oauth:jwk-thumbprint:sha-256:_Fpfe27AuGmEljZE9s2lw2UH-qrZLRFNrWbJrWIe4SI",
+  "verifiableCredential": [
     {
-      "id": "#credential-issuance",
-      "type": "JsonWebKey",
-      "controller": "did:web:vendor.example",
-      "publicKeyJwk": {
-        "kty": "EC",
-        "crv": "P-256",
-        "alg": "ES256",
-        "x": "MYvnaI87pfrn3FpTqW-yNiFcF1K7fedJiqapm20_q7c",
-        "y": "9YEbT6Tyuc7xp9yRvhOUVKK_NIHkn5HpK9ZMgvK5pVw"
+      "@context": [
+        "https://www.w3.org/ns/credentials/v2"
+      ],
+      "type": [
+        "VerifiableCredential"
+      ],
+      "issuer": "https://issuer.example/issuers/68",
+      "validFrom": "2023-06-07T21:14:14.148Z",
+      "credentialSubject": {
+        "id": "https://subject.vendor.example"
       }
     },
-    {
-      "id": "#key-agreement",
-      "type": "JsonWebKey",
-      "controller": "did:web:vendor.example",
-      "publicKeyJwk": {
-        "kty": "OKP",
-        "crv": "X25519",
-        "alg": "ECDH-ES+A128KW",
-        "x": "qLZkSTbstvMWPTivmiQglEFWG2Ff7gNDVoVisdZTr1I"
-      }
-    }
-  ],
-  "authentication": ["#subject-authenticaton"],
-  "assertionMethod": ["#credential-issuance"],
-  "keyAgreement": ["#key-agreement"]
-}
-            
-      </pre>
-     
+    "https://vendor.example/credentials/42",
+    "did:example:123",
+    "urn:uuid:01ec9426-c175-4e39-a006-d30050e28214",
+    "urn:ietf:params:oauth:jwk-thumbprint:sha-256:_Fpfe27AuGmEljZE9s2lw2UH-qrZLRFNrWbJrWIe4SI",
```

Comment:
Curious how these 3 items are Verifiable Credentials?

Reply:
the are not, but this RDF term defines what `verifiableCredential` means when it is in a presentation.

https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L84

As you can see, its not defined in the vocabulary... cc @iherman ...

https://www.w3.org/2018/credentials#verifiableCredential

However, you can tell from the `@context` the above syntax is intended, see:

```   "verifiableCredential": {
          "@id": "https://www.w3.org/2018/credentials#verifiableCredential",
          "@type": "@id", // this tells you `id` values are acceptable
          "@container": "@graph" // this tells you members are separate graphs.
        },
```